### DISCUSSION TITLE
Editing rego plate on A32NX

### DISCUSSION THREAD

**Discussion by matt2789** [flybywiresim/aircraft#4360] [Help]
Hi Guys,

Sorry if this has been asked before, im very new to editing stuff. I have downloaded GIMP and have the file with rego open, I just cant figure out how to actually edit the text or do i just have to create a new text box on that layer? 

Sorry if this is a stupid question!

**Comment by ExampleWasTaken** [comment]
Have a look a [this video](https://youtu.be/Aa_JRHd33eo) from British Avgeek.

**Reply by matt2789** [reply]
thanks heaps!
